cmd/zed/relationship.go

```diff
@@ -70,25 +77,121 @@ var readCmd = &cobra.Command{
 	RunE:              readRelationships,
 }
 
-func readRelationships(cmd *cobra.Command, args []string) error {
+var bulkDeleteCmd = &cobra.Command{
+	Use:               "bulk-delete <resource_type:optional_resource_id> <optional_relation> <optional_subject_type:optional_subject_id#optional_subject_relation>",
+	Short:             "bulk delete Relationships",
+	Args:              cobra.RangeArgs(1, 3),
+	PersistentPreRunE: persistentPreRunE,
+	RunE:              bulkDeleteRelationships,
+}
+
+func bulkDeleteRelationships(cmd *cobra.Command, args []string) error {
+	configStore, secretStore := defaultStorage()
+	token, err := storage.DefaultToken(
+		cobrautil.MustGetString(cmd, "endpoint"),
+		cobrautil.MustGetString(cmd, "token"),
+		configStore,
+		secretStore,
+	)
+	if err != nil {
+		return err
+	}
+	log.Trace().Interface("token", token).Send()
+
+	client, err := authzed.NewClient(token.Endpoint, dialOptsFromFlags(cmd, token.ApiToken)...)
+	if err != nil {
+		return err
+	}
+
+	request, err := buildReadRequest(cmd, args)
+	if err != nil {
+		return err
+	}
+
+	if !cobrautil.MustGetBool(cmd, "force") {
```

Comment:
maybe we should have a different flag for skipping the read preview
`estimate-count=false` or something

Reply:
Do you think a prompt without the count is useful though?

Reply:
Yes, if I know it's huge.

Reply:
I'll add it, but I don't see many using it